openslides_backend/models/fields.py

```diff
@@ -100,7 +100,12 @@ def get_schema(self) -> Schema:
 
 
 class JSONField(TextField):
```

Comment:
I think this no longer should inherit from `TextField`, right? I think the initial thought was that it should represent the json-encoded string, but this is obsolet now.

Reply:
Yes and done